docker/app/Dockerfile

```diff
@@ -53,13 +53,16 @@ RUN apt-get remove -y gnupg2
 RUN mv $PHP_INI_DIR/php.ini-production $PHP_INI_DIR/php.ini
 # had to add /wait into prod image so the prod image could be run through E2E tests.
 COPY --from=sillsdev/web-languageforge:wait-latest /wait /wait
+RUN apt-get update && apt-get -y install tini && rm -rf /var/lib/apt/lists/*
```

Comment:
This is a good.  If we standardize on `tini` then we should just put this in the `web-languageforge:base-php` image which is where it belongs, I think.

Reply:
I find the optimization `rm -rf /var/lib/apt/lists/*` annoying when it appears in more than one place in the docker file.  We should be able to do this once at the end of the Dockerfile and save image build time.

Reply:
Agree about putting it in base-php; I'll do that on Monday.

As for the `rm -rf` optimization, it's for layer size; if an earlier /var/lib/apt/lists is allowed to stand, then its layer is much larger and all subsequent layers incorporate those files, meaning that the later `rm -rf` doesn't actually do anything to reduce the total image size. So if you want the /var/lib/apt/lists files to be absent from the final image, reducing total image size, then the `rm -rf` command needs to appear in the same line as every `apt-get update`.